documentation/source/equilibria/equilibria.rst

```diff
@@ -772,35 +772,102 @@ current optimiser.
+
+Additionally, when the current value of a Circuit is set, the current value of all
+coils in the Circuit is set to the same value.
+
+Using Circuits in an optimisation
+*********************************
+
+When using Circuits in an equilibria optimisation,
+the reduced current state used by the optimiser is expanded in the following way,
+when calling each constraint function:
+
+.. code-block:: python
+
+   if coilset._contains_circuits:
+         # wrap the constraint function
+         @functools.wraps(f.f_constraint)
+         def wrapped_f_c(x, f=f):
+            return f.f_constraint(coilset._opt_currents_expand_mat @ x)
+
+ f_c = wrapped_f_c
+
+         if df_c is not None:
+            # wrap the derivative function
+            @functools.wraps(f.df_constraint)
+            def wrapped_df_c(x, f=f):
+ df_res = f.df_constraint(coilset._opt_currents_expand_mat @ x)
+               return df_res @ coilset._opt_currents_expand_mat
+
+ df_c = wrapped_df_c
+
+The ::pycode:`_opt_currents_expand_mat` property of a CoilSet is a matrix with 1'S in the
+columns corresponding to the coils in each Circuit and 0's elsewhere. Refer to
+:py:meth:`~bluemira.equilibria.optimisation.base._make_numerical_constraints`.
+
+When implementing a constraint, no special consideration is needed for the shape of the
+state vector passed into the constraint. It must be assumed that the full current vector
+(i.e. length equal to the number of coils) is passed into the constraint function.
+
+However, special care is needed when implementing a figure of merit (FoM),
+in a  :py:class:`~bluemira.equilibria.optimisation.problem.base.CoilsetOptimisationProblem` (COP)
+class that operates on the current vector.
+
+FoMs in COPs are implemented in a more bespoke fashion than constraints are and thus are
+more flexible. Usually, one has access to the full Equilibrium object when
+implementing the FoM function and may use the ``_opt_currents_expand_mat``
+from a CoilSet
+
+Refer to :py:class:`~bluemira.equilibria.optimisation.problem._tikhonov.Tikhonov` for an example of a FoM
+that uses the ``_opt_currents_expand_mat`` CoilSet property.
+
+.. Note::
+ It should be noted that the derivative vector returned by the ``df_constraint`` function
+ is matmul'd by the ``_opt_currents_expand_mat`` matrix, which effectively sums the
+ derivatives of the coils in each Circuit. This is valid you are taking the derivative
+ with respect to the same current value, per coil in each circuit.
+
+ This converts the derivative vector into the correct shape the optimiser expects.
+
+ Refer to this pull request for more (`PR #3292 <https://github.com/Fusion-Power-Plant-Framework/bluemira/pull/3292>`_.).
+
+SymmetricCircuit
+*****************
+
+In the case where one wants to solve a double null equilibrium, it is expected that the
+equilibrium should be symmetrical about :math:`z=0`.
+
+The ``SymmetricCircuit`` class has been developed as a special case of a ``Circuit``,
```

Comment:
it'd be good to reference both these classes at least in the first instance here